modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -45,6 +45,7 @@ function () {
 
 		add_action( 'admin_menu', array( $this, 'reregister_menu_items' ), 99999 );
 		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'dequeue_scripts' ), 20 );
```

Comment:
Just double-checking the script is still needed in the front-end?

Reply:
Couldn't find a usecase in my Jetpack dev environment but opened an atomic one and seems that we should get rid of these scripts in the frontend too.